codalab/model/bundle_model.py

```diff
@@ -1052,6 +1053,15 @@ def do_update(connection):
             with self.engine.begin() as connection:
                 do_update(connection)
 
+    def get_bundle_dependencies(self, uuid):
```

Comment:
This just allows the `get_target_info` function to make a lighter DB call to fetch only the dependencies, rather than fetching everything about the bundle.